game/src/contrib/components/StatsComponent.java

```diff
@@ -10,13 +10,19 @@
  * Add resistances and vulnerabilities for {@link contrib.utils.components.health.DamageType}s to
  * the associated entity.
  *
+ * <p>Resistances refer to the ability of an entity to mitigate or reduce the damage taken from
+ * specific {@link DamageType}.
+ *
+ * <p>Vulnerabilities indicate specific weaknesses of an entity, making them more susceptible to
+ * increased damage or negative effects from certain {@link DamageType}.
+ *
  * <p>To set resistances and vulnerabilities for a damage type, use {@link #multiplier(DamageType,
- * float)}. For resistances, use a float less than 1, for vulnerabilities use a float greater than
- * 1.
+ * float)}. For resistances, use values less than 1.0, for vulnerabilities use values greater than
+ * 1.0.
```

Comment:
Nochmal ne blöde Frage von der Seite ;)

Es ist ein Multiplikator, d.h. ich kann auch locker negative Werte reintun. Aber dann kehrt sich alles einmal um, d.h. aus Resistances werden Vulnerabilities und umgedreht, oder?

Soll das so? Oder sollte man gezielt zwei Multiplikatoren anbieten, die beide stets positiv bleiben (per Zwangs-`Math.abs()`) und wo man gezielt von außen hoch- und runterdrehen kann - im Bereich zw. 0 und 100 für beide und intern wird das entsprechend umgerechnet? Oder wird das verwirrender für die Anwender?

Reply:
Uff... dir fallen Sachen auf.

> Es ist ein Multiplikator, d.h. ich kann auch locker negative Werte reintun. Aber dann kehrt sich alles einmal um, d.h. aus Resistances werden Vulnerabilities und umgedreht, oder?

Ne, eigentlich wird dann aus Schaden ein Heilwert.
Schadenswert  * -faktor =  -Schaden 


Auf anhieb fällt mir dazu ein, dass ich ja vielleicht "Super-Resistent" sein könnte, ein Feuermonste könnte von Feurschaden z.B geheilt werden, dann wäre ein negativer Faktor sinnvoll. 
Müsste aber mit in die Dokumentation.

Reply:
Habe es an die entsprechenden stellen geschrieben.

Reply:
@AMatutat Willst Du nicht lieber noch ein Ticket aufmachen, damit das nochmal vernünftig gemacht wird? Jetzt ist es zwar dokumentiert, aber schön ist es irgendwie noch nicht.